sympy/calculus/tests/test_util.py

```diff
@@ -545,3 +545,10 @@ def test_issue_16469():
 
 def test_issue_18747():
     assert periodicity(exp(pi*I*(x/4+S.Half/2)), x) == 8
+
+def test_AccumBounds_hooks():
+    a = AccumBounds(1,2)
+    assert Mul(2,a, evaluate=True) == 2*a
```

Comment:
It would be better if this test asserted what these are equal to like:
```
    assert Mul(2,a, evaluate=True) == 2*a == AccumBounds(2, 4)
```

Reply:
Fixed it.